azimuth/routers/v1/model_performance/outcome_count.py

```diff
@@ -78,10 +78,12 @@ def get_outcome_count_per_filter(
     task_manager: TaskManager = Depends(get_task_manager),
     dataset_split_manager: DatasetSplitManager = Depends(get_dataset_split_manager),
     pipeline_index: int = Depends(require_pipeline_index),
+    without_postprocessing: bool = False,
```

Comment:
Oups, you missed this one, that's why the distribution bars in the control panel don't change, as the `withoutPostprocessing` query parameter is ignored.
```suggestion
    without_postprocessing: bool = Query(
        False, title="Without Postprocessing", alias="withoutPostprocessing"
    ),
```

Reply:
Ok, I did it in 7049af2e5bbc0d197431c467d7ef69ec6eaf847f

Reply:
oops! I added myself in the code..